_pages/about.markdown

```diff
@@ -13,8 +13,8 @@ header:
 feature_row:
   - image_path: /assets/images/about/deployment1.jpg
     alt: "Deployment on IPR"
-    title: "TermIt deployment on IPR"
-    excerpt: "TermIt runs on IPR since 2020. It was originally used to describe Metropolitan plan and help to relate it to the legislation. The main problem to solve is that experts from different domains uses different terminology, and all of them must contribute. Nowadays, TermIt serves as an intermediary for many other projects besides Metropolitan plan."
+    title: "TermIt deployment on Institute for Planning and Development of Prague (IPR)"
+    excerpt: "TermIt runs on IPR since 2021. TermIt was originally created in the project OPPPR and IPR tested its usability -- to describe Metropolitan plan and help to relate it to the legislation. The main problem to solve is that experts from different domains uses different terminology, and all of them must contribute. Nowadays, TermIt serves for systemization of terminology in other projects, e.g. Uniform Interchangeable Format of the Digital Technical Map (JVF DTM) and Building Intelligence Model (BIM)."
```

Comment:
Here are some identification details about the project: https://starfos.tacr.cz/en/project/UH0367